Task_DEV-9/Task_DEV-9/Tests/LoginYandexTests.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System;
+using Task_DEV_9.Yandex;
+using NUnit.Framework;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+
+namespace Task_DEV_9.Tests
+{
+    [TestFixture]
+    public class LoginTestsYandex
+    {
+        private IWebDriver driver;
+
+        [SetUp]
+        public void Start()
+        {
+            this.driver = new ChromeDriver();
+            this.driver.Navigate().GoToUrl("https://yandex.by");
+            this.driver.Manage().Timeouts().ImplicitWait = TimeSpan.FromSeconds(5);
+        }
+
+        [Test]
+        public void Login_CorrectId_Proceed()
+        {
+            var startPage = new YandexLogin(this.driver);
+            var email = "dima.kupeshka@yandex.by";
```

Comment:
it is better to send credentials like parameters or just get them from some file of resources